modules/osd-create-cluster-ccs.adoc

```diff
@@ -134,6 +134,16 @@ endif::osd-on-gcp[]
 .. Select a cluster version from the *Version* drop-down menu.
 .. Select a cloud provider region from the *Region* drop-down menu.
 .. Select a *Single zone* or *Multi-zone* configuration.
++
+ifdef::osd-on-gcp[]
+.. Optional: Select *Enable Secure Boot for Shielded VMs* to use Shielded VMs when installing your cluster. For more information, see link:https://cloud.google.com/security/products/shielded-vm[Shielded VMs].
++
+[IMPORTANT]
+====
+To successfully create a cluster, you must select *Enable Secure Boot support for Shielded VMs* if your organization has the policy constraint 'constraints/compute.requireShieldedVm' enabled. For more information regarding GCP organizational policy constraints, see link:https://cloud.google.com/resource-manager/docs/organization-policy/org-policy-constraints[Organization policy constraints].
```

Comment:
"constraints/compute.requireShieldedVm" should be marked up somehow, either backticks or maybe bold. Looking elsewhere in the repo, it seems that other policy constraints have used backticks:

```suggestion
To successfully create a cluster, you must select *Enable Secure Boot support for Shielded VMs* if your organization has the policy constraint `constraints/compute.requireShieldedVm` enabled. For more information regarding GCP organizational policy constraints, see link:https://cloud.google.com/resource-manager/docs/organization-policy/org-policy-constraints[Organization policy constraints].
```

There are 2 other instances of this in the PR that should likewise be marked up.